docs/client.rst

```diff
@@ -174,6 +181,14 @@ build does not include the C++ examples.
 The MSBuild.exe may need to be invoked twice for a successfull
 build.
 
+A wheel file may fail to generate if the python environment is not set up 
+correctly. The windows build environment will not fail correctly if there is a 
+failure when running the python script. To verify that wheel file is executed
```

Comment:
".. is generated correctly..."

This is confusing. Is the command actually generating the wheel file? If so you should state more clearly that the command is generating the wheel (not just verifying it), and also say where the resulting wheel file can be found.